content/pop-recovery.md

```diff
@@ -63,7 +63,9 @@ If the existing install is encrypted, please see the [encrypted disk](#encrypted
 
 If the existing OS install needs to be repaired, the installer application should be closed. An app menu is located in the top-left of the screen with the name of the currently running application (in this case: "Install Pop!\_OS"). Click on the app menu and select `Quit`. Alternatively, you can use the installer app to select keyboard and language settings, and then click the `Try Demo Mode` button in the lower-left corner of the install page.
 
->**Note:** Do not choose any install or repair options, as this could result in data loss.
+>**Note:** Do not choose any install or repair options, as this could result in data loss. 
+
+>**Note:** Also, once you have setup or accessed any of the filesystems, you will need to boot to the recovery environment anew to use any of the install options.
```

Comment:
How about, `**Note:** If you have accessed any filesystems and intend to install or refresh afterwards, the machine should be rebooted into a fresh recovery environment to clear any created mount points.` instead? Still feels a bit clumsy to me, but it feels better to lead with the situation in which the note is important for the reader.

Reply:
I like this update better. Looking at the Live version of this change looks good to me.